assets/styles/cgu.scss

```diff
@@ -0,0 +1,103 @@
+.section-1-cgu {
+    padding-top: 5rem;
+}
+
+.grid-cgu {
+    padding:2em;
+    background-color:white;
+    display: grid; 
+    grid-auto-columns: 1fr; 
+    grid-template-columns: 1fr 1fr; 
+    grid-template-rows: 1fr 1fr; 
+    gap: 1em 1em; 
+}
+
```

Comment:
The following code is the same as the RGPD scss. I think you don't need it (i tried without and the css words the same)